src/libraries/JBFees.sol

```diff
@@ -7,29 +7,27 @@ import {JBConstants} from "./../libraries/JBConstants.sol";
 
 /// @notice Fee calculations.
 library JBFees {
-    /// @notice Returns the amount of tokens to pay as a fee out of the specified `amount`.
-    /// @dev The resulting fee will be `feePercent` of the REMAINING `amount` after subtracting the fee, not the full
-    /// `amount`.
-    /// @param amount The amount that the fee is based on, as a fixed point number.
+    /// @notice Returns the amount of tokens to pay as a fee relative to the specified `amount`.
```

Comment:
nit: this is slightly confusing, but im also not sure how to word it so it isn't confusing.